mangaki/mangaki/models.py

```diff
@@ -81,13 +85,15 @@ class Work(models.Model):
 
     # Some of these fields do not make sense for some categories of works.
     genre = models.ManyToManyField('Genre')
+    tags = models.ManyToManyField('Tag', through="TaggedWork")
     origin = models.CharField(max_length=10, choices=ORIGIN_CHOICES, default='', blank=True)
     nb_episodes = models.TextField(default='Inconnu', max_length=16, blank=True)
     anime_type = models.TextField(max_length=42, blank=True)
     vo_title = models.CharField(max_length=128, blank=True)
     manga_type = models.TextField(max_length=16, choices=TYPE_CHOICES, blank=True)
     catalog_number = models.CharField(max_length=20, blank=True)
     anidb_aid = models.IntegerField(default=0, blank=True)
+    have_anidb_aid = models.BooleanField(default=False)
```

Comment:
Je réitère ma remarque : 

Pour avoir la possibilité de choisir entre ceux qui ont / n'ont pas de anidb_aid, tu peux utiliser un filtre custom (cf ici https://docs.djangoproject.com/en/1.9/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_filter ).

Écrire une classe héritant de `SimpleListFilter` pour ça plutôt que d'ajouter un champ dans le modèle.